vector/src/main/java/im/vector/app/features/home/room/detail/TimelineViewModel.kt

```diff
@@ -1165,6 +1169,9 @@ class TimelineViewModel @AssistedInject constructor(
                 summary.inviterId?.let { inviterId ->
                     session.getRoomMember(inviterId, summary.roomId)
                 }?.also {
+                    if (initialState.isInviteAlreadyAccepted) {
+                        handle(RoomDetailAction.AcceptInvite)
+                    }
```

Comment:
I would move these new lines in the init block of the ViewModel, also you could call directly the fun `handleAcceptInvite()`. WDYT?